src/index.html

```diff
@@ -9,6 +12,23 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img src="images/logo.png" alt="logo">
+      <nav>
+        <ul>
+          <li><a href="Apple" class="is-active">APPLE</a> </li>
+          <li><a href="Samsung" class=" ">SAMSUNG</a></li>
+          <li><a href="Smarthphones" class=" ">SMARTPHONES</a></li>
+          <li>
+            <a href="Laptops & Computers" class="frth-elem"
```

Comment:
no needs to use such classes as `rth-elem` or `last-element`
all links should have the same class `nav__link`